### ISSUE TITLE
improve loss function performance

### ISSUE THREAD

**Issue by sandhyacv** [vaibhav-prasad707/bias-detection-neutrosophic-loss#6]
possible initial approaches:

1. label smoothing

2. task-specific normalizations

3. dynamic task-weighting for loss

4. change output activation from softmax

5. regularization on probability distribution

6. class-weighting to mitigate class imbalance

7. adjust weights: task-specific, dynamic weighting

8. change the final loss equation: exponentiate/square F & I for penalizing aggressively, multiple F & I, etc.
